src/table/mod.rs

```diff
@@ -418,7 +439,7 @@ impl Table {
         for _ in 0..self.players.len() {
             // set the next dealer index by finding the next alive player
             self.dealer_button_index += 1;
-            if self.dealer_button_index + 1 >= self.get_player_count() {
+            if self.dealer_button_index >= self.get_player_count() {
```

Comment:
Was the last player just never dealer before?

Reply:
Yeah think so… oopsie.